### PR TITLE
Now Command can be passed in multiple attributes

### PR DESCRIPTION
**Pull request by stek29**
Also fixed typo (tab after last `fi`)

### CODE REVIEW

File: marathon [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 fi
+
+command="$*"
 # zenity --info --text "$command"; # useful for debug
 
 runningInstancesOfExec=$(ps -a -x -o command | grep -v grep | grep -v defunct | grep -v marathon | grep -c $command);
 
 if [ $runningInstancesOfExec -gt 0 ]; then
-	if [[ $1 == "--toggle" ]]; then
+	if [[ toggle == "1" ]]; then
```
Comment (ronjouch) [inline marathon]: @stek29 typo: `toggle` should be `$toggle`. By the way, what does your patch enable?

Comment (ronjouch) [inline marathon]: @stek29 typo: `toggle` should be `$toggle`. By the way, what does your patch enable? (What's your use case where you'd call marathon with multiple arguments?)

EDIT: I'd guess you're having multiple sessions of the same app, e.g. `gvim -S work` and `gvim -S home`, and you want a keystroke per app even if it's the same binary. Right?

**Comment by stek29** [comment]
Yes, you are right
I know that I can use `marathon --toggle "gvim -S home"` instead of `marathon --toggle gvim -S home`
But when I used it first time I was a little bit confused, so I desided to change script so second command works too

And about last commit:
Sha-Bang is `/bin/sh`, but on Ubuntu and many other distros `sh` is link to `dash`, not to `bash`
But `[[ ]]` and `[ == ]` work only in `bash`, and do not work in other shells like `dash` or `zsh`
You can read more [here](http://mywiki.wooledge.org/Bashism)

**Comment by ronjouch** [comment]
@stek29 hi!

> Yes, you are right. I know that I can use `marathon --toggle "gvim -S home"` instead of `marathon --toggle gvim -S home`. But when I used it first time I was a little bit confused, so I desided to change script to make second command work

Great, willing to merge this :+1: 

> And about last commit: Sha-Bang is `/bin/sh`, but on Ubuntu and many other distros sh is link to `dash`, not to `bash` . But `[[ ]]` and `[ == ]` work only in bash, and do not work in other shells like `dash` or `zsh`. You can read more [here](http://mywiki.wooledge.org/Bashism).

Thanks for pointing out the bashism, I'm using zsh myself. Bug the problem is that currently your code not including the `$` does not work for me. Looking at your wiki link, I think that we'll both be happy with:

* Line 4: `if [ "$1" == "--toggle" ]; then`
* Line 17: `	if [ "$toggle" == "1" ]; then`

If you confirm, pls make these changes and update your fork branch with a `git commit -a --amend && push force origin master`, and I'll merge. Thanks again!

**Comment by ronjouch** [comment]
@stek29 oh my, didn't see you did exactly this. Merging, merci beaucoup.

**Pull request merged**